zmon_worker_monitor/zmon_worker/notifications/twilio.py

```diff
@@ -28,20 +28,24 @@ def notify(cls, alert, *args, **kwargs):
 
         repeat = kwargs.get('repeat', 0)
         oauth2 = kwargs.get('oauth2', True)
+        key = kwargs.get('key', cls._config.get('notification.service.key', None))
```

Comment:
Same here: why do we need this config if `OAUTH2_ACCESS_TOKENS` can be specified already?